.coafile

```diff
@@ -0,0 +1,26 @@
+
+[all]
+bears = LineCountBear, FilenameBear
+files = **.py, **.yaml, **.toml, **.rst, **.md
+file_naming_convention = snake
+ignore = **/__pycache__/**, **/__pycache__, __pycache__, __pycache__/**, **/*.pyc, *.pyc
+max_line_length = 120
+max_lines_per_file = 2000
+
+[all.python]
```

Comment:
Thanks @harshad16 ! Just a note to the python liners, I don't think that they are necessary, I don't see this being a python package. WDYT @fridex? (Anyway, it doesn't make much difference to have them there or not to)

Reply:
+1, this is probably a copy-paste configuration.

BTW we could think about organization-wide style and configure linters accordingly (so we don't need to place config files in each repo).

Reply:
Yep, I think that the template repo would be a nice thing to have.

Reply:
Or we could be more adventurous and let Kebechet generate the configuration according to the language of the repo? @fridex

Reply:
I would probably move this to zuul CI so we do not maintain these files all over the place but rather have one configuration which is automatically used in CI on a repo.

Reply:
Ack.

Reply:
@CermakM I had just copy-pasted with respect to other repos, I will remove it if python is not being used here. :)